ikarus/finiteElements/mechanics/nonLinearElasticityFE.hh

```diff
@@ -114,26 +143,134 @@ namespace Ikarus {
           const auto u = uFunction.evaluate(quadPos);
 
           // Value of the Neumann data at the current position
-          auto neumannValue = neumannBoundaryLoad_(toEigen(intersection.geometry().global(curQuad.position())), lambda);
-
+          auto neumannValue
+              = (*neumannBoundaryLoad_)(toEigen(intersection.geometry().global(curQuad.position())), lambda);
           energy -= neumannValue.dot(u) * curQuad.weight() * integrationElement;
         }
       }
-
       return energy;
     }
 
+    void calculateMatrix(const FERequirementType& par, typename Traits::MatrixType& K) const {
+      using namespace Dune::DerivativeDirections;
+      using namespace Dune;
+      const auto eps = getStrainFunction(par);
+      const auto geo = this->localView().element().geometry();

```

Comment:
`using namespace Dune::DerivativeDirections;` allows the simplification below `Dune::DerivativeDirections::gridElement` --> `gridElement`